src/aat/java/uk/gov/hmcts/ccd/datastore/tests/AATHelper.java

```diff
@@ -60,7 +60,7 @@ public String getCaseworkerAutoTestPassword() {
     }
 
     public String getDefinitionStoreUrl() {
-        return Env.require("CCD_DEFINITION_STORE_URL");
+        return Env.require("DEFINITION_STORE_HOST");
```

Comment:
Reusing an existing environment variable

Reply:
@Nayab this will impact your local tests, please replace CCD_DEFINITION_STORE_URL with DEFINITION_STORE_HOST in your IDE environment variables